Curtains/Curtains.Infrastructure/Migrations/20220901113709_InitCreate.Designer.cs

```diff
@@ -0,0 +1,702 @@
+﻿// <auto-generated />
+using System;
+using Curtains.Infrastructure.Models;
```

Comment:
There seems to be an issue, maybe we should change this dependency to <code>using Curtains.Domain.Models;</code>